wee_slack.py

```diff
@@ -2613,6 +2613,23 @@ def unhtmlescape(text):
                .replace("&amp;", "&")
 
 
+def rgb2term_prefix(color):
+    if color:
+        try:
+            # Strangely enough color_rgb2term requires hex color as string
+            # representation of 0xRRGGBB in base-10...
+            color_int = str(int(color.lower(), 16))
+            color_code = w.info_get('color_rgb2term', color_int)
+        except ValueError:
+            # In case color is not valid hex representation, just use it as is
+            # weechat should gracefuly handle most basic colors (yellow, blue,
+            # ...) or fall back to defaults
+            color_code = color.lower()
+
+        return w.color(color_code)
```

Comment:
TBH this whole part could use weechat format expressions, as I actually personally prefer it this way.
I was thinking about adding enum-like configuration for that (`prefix`/`message`/`disable`) as a temporary solution... (but got into doing other things...)